plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTableColumn.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableColumn;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.Property;
+
+public class CubridTableColumn extends GenericTableColumn
+{
+    public CubridTableColumn(GenericTableBase table, JDBCResultSet dbResult) throws DBException
+    {
+        super(table);
+        if (dbResult != null) {
+        	long columnSize = 0;
+            typeName = JDBCUtils.safeGetString(dbResult, "Type");
+            if (typeName != null && typeName.contains("(")) {
+                columnSize = Long.parseLong(typeName.replaceAll("[^0-9]", ""));

```

Comment:
If the value is empty, an exception may occur.
e.g) ENUM('Y', 'N')

Reply:
Well noted, I will fix this issues. However, currently This PR is covered data type following Generic data type. So, Enum type will not display in UI, can only create using script.

Reply:
We are planning to modify the JDBC Driver and
After modification, it appears as below.
![image](https://github.com/longhaseng52/dbeaver/assets/68524199/5ea62290-590f-4e9d-9b44-e20e2482a2f4)

Additionally, an exception may appear when retrieving data that has already been entered.

Reply:
I've fixed the error when retrieving data in the last commit.